index.js

```diff
@@ -92,6 +96,26 @@ class Conf {
 		this.store = plainObject();
 	}
 
+	migrate(options, expectedVersion) {
```

Comment:
It should be marked internal.

```suggestion
	_migrate(options, expectedVersion) {
```